images/packages/binaries/libgsasl/werf.inc.yaml

```diff
@@ -0,0 +1,74 @@
+---
+image: {{ $.ImageType }}/{{ $.ImageName }}
+final: false
+fromImage: builder/scratch
+import:
+- image: {{ $.ImageType }}/{{ $.ImageName }}-builder
+  add: /out
+  to: /libgsasl
+  includePaths:
+    - usr/bin/gsasl
+    - usr/lib64/libgsasl.so
+    - usr/lib64/libgsasl.so.18
+    - usr/lib64/libgsasl.so.18.0.0
+  before: setup
+- image: {{ $.ImageType }}/{{ $.ImageName }}-builder
+  add: /out
+  to: /libgsasl-devel
+  includePaths:
+    - usr/include/gsasl-mech.h
+    - usr/include/gsasl-version.h
+    - usr/include/gsasl.h
+    - usr/lib64/libgsasl.la
+    - usr/lib64/pkgconfig/libgsasl.pc
+    - usr/share/info/gsasl.info
+  before: setup
+
+---
+{{- $version := get $.Package $.ImageName }}
+{{- $gitRepoUrl := "gsasl/gsasl.git" }}
+
+{{- $name := print $.ImageName "-dependencies" -}}
+{{- define "$name" -}}
+packages:
+- gcc git make libtool gettext-tools
+- wget patch gperf help2man texinfo gengetopt
```

Comment:
Building doc ans nls are disabled with options:

```
 --disable-doc \
      --disable-nls
```

Can you check if build is succeeded without  gettext-tools, help2man and texinfo packages?

Reply:
Oh, I remember, --disable-doc is not disable man build. Now worry here.